Fika.Core/Coop/Utils/FikaBackendUtils.cs

```diff
@@ -31,29 +26,31 @@ public static class FikaBackendUtils
         public static int RemotePort;
         public static int LocalPort = 0;
         public static bool IsHostNatPunch = false;
-        private static string groupId;
-        private static string raidCode;
+        private static string _serverId;
+        private static string _raidCode;
```

Comment:
```suggestion
        private static string raidCode;
```
I'm trying to steer away from `_private` variables to keep the code base consistent.